src/main/java/com/actionlistapi/pojo/_Links.java

```diff
@@ -0,0 +1,42 @@
+package com.actionlistapi.pojo;
+
+public class _Links {
```

Comment:
This class isn't necessary if you use Spring HATEOAS: http://projects.spring.io/spring-hateoas/